lisk-dex-electron/src/config/Configuration.js

```diff
@@ -0,0 +1,91 @@
+import { getClient } from '../API';
+import { Number, String, Array, Record, Dictionary } from 'runtypes';
```

Comment:
It's better to not overwrite JavaScript native types like `Number`, `String` and `Array` in case we need to use the native types (can be confusing).

It would be better to use a short variable like `rt` then do `rt.Number`. It's  a bit more repetitive but clearer.